docs/HeliumParamSpecs.md

```diff
@@ -0,0 +1,161 @@
+         * Response Body: List of movies filtered by year as [{}, {},..], , empty [] when no results
+    * Invalid input - empty, foo, values not in valid range
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid year parameter
+
+
+* Name : rating
+* Type: double
+* Parameter Validation:
+    * Valid input range : [0.0, 10.0]
+    * Round off more than single decimal to a single decimal. Example: round off 7.253896 to 7.2.
+         * Status : 200
+         * Content-Type: application/json
+         * Response Body: List of movies filtered by rating as [{}, {},..], empty [] when no results
+    * Invalid input - empty, foo, values not in valid range [<0.0 , >10.0]
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid rating parameter
+
+
+* Name : actorid
+* Type: string
+* Parameter Validations
+    * Valid input : Wellformed actorid rule - starts with 'nm' followed by upto a 9 digit integer converted to string. Example 'nm9877392'
+         * Status : 200
+         * Content-Type: application/json
+         * Response Body: List of movies filtered by actorid as [{}, {},..] , empty [] when no results
+    * Invalid input - empty, foo, values not conforming to wellformed actorid rule.  Example: 'nm1234', 'ab1234'
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid actorid parameter
+
+
+* Name : genre
+* Type: string
+* Parameter Validations:
+    * Valid input - genre length >= 3. Example 'War' , 'Horror'
+         * Status : 200
+         * Content-Type: application/json
+         * Response Body: List of movies filtered by actorid as [{}, {},..] , empty [] when no results
+    * Invalid input - empty, genre length < 3 characters
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid genre parameter
+
+
+* Name : pagesize
+* Type: integer
+* Parameter Validation:
+    * Default value: 100
+    * Valid input range : [1, 1000]
+         * Status : 200
+         * Content-Type: application/json
+         * Response Body: List of movies filtered by pagesize as [{}, {},..], empty [] when no results
+    * Invalid input - Non integers are invalid. Example : 100.23
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid pagesize parameter
+
+
+* Name : pagenumber
+* Type: integer
+* Parameter Validations:
+    * Default value: 1
+    * Valid input range : [1, 10000]
+         * Status : 200
+         * Content-Type: application/json
+         * Response Body: List of movies filtered by pagenumber as [{}, {},..], empty [] when no results
+    * Invalid input - Non integers are invalid. Example : 100.23
+         * Status : 400
+         * Content-Type: text/plain
+         * Response Body: Invalid pagenumber parameter
+
+
+## Multiple Error Scenarios
+
+* Add each error to the errors array in alphabetical order.
```

Comment:
Maybe move this "Multiple Error Scenarios" section to the bottom, or at least after "API Parameters",  as is when rendered, it looks like the next "API Parameters" section is a sub section of this.